TS00-Brief-Tabular-Time-Series-In-R.Rmd

```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
-title: "Lesson 00: Brief Time Series in R - Simple Plots with qplot & as.Date
-Conversion"
+title: "Lesson 00: Intro to Time Series Data in R - Managing Date / Time Formats
+Simple Plots using GGPLOT2"
```

Comment:
Intro to Time Series Data in R - Managing Date / Time Format and Simple Plots using ggplot2